docs/plot/legend.rst

```diff
@@ -0,0 +1,62 @@
+.. currentmodule:: gwpy.plot
+
+.. _gwpy-plot-legend:
+
+######################
+Custom legends in GWpy
+######################
+
+GWpy overrides the default :class:`~matplotlib.axes.Axes` class with one that
+uses a different default legend handler for line plots.
+This means that, by default, a lines in a legend will be thicker than on a
+standard matplotlib figure:
+
+.. plot::
+   :include-source:
+   :context: reset
+
+   >>> import gwpy  # <- import anything from gwpy
+   >>> from matplotlib import pyplot
+   >>> fig = pyplot.figure()
+   >>> ax = fig.gca()
+   >>> ax.plot(range(10), label='My data')
+   >>> ax.legend()
+   >>> fig.show()
+
+This can be disabled by passing an empty ``handler_map`` to the
+:meth:`~matplotlib.axes.Axes.legend` method:
+
+.. plot::
+   :include-source:
+   :context: close-figs
+
+   >>> fig = pyplot.figure()
+   >>> ax = fig.gca()
+   >>> ax.plot(range(10), label='My data')
+   >>> ax.legend(handler_map=None)
+   >>> fig.show()
+
+Similarly, you can implement your own custom legend handler and overwrite
+things yourself, below is a simple example, but for more details see
```

Comment:
@duncanmmacleod, this probably comes down to style, but I think the last two clauses ought to be a separate sentence:

```suggestion
things yourself. Below is a simple example, but for more details see
```